Tasks/VsTest/distributedtest.ts

```diff
@@ -54,208 +56,81 @@ export class DistributedTest {
-        inputDataContract.TiaBaseLineBuildIdFile = this.dtaTestConfig.tiaConfig.baseLineBuildIdFile;
-        inputDataContract.VsVersion = this.dtaTestConfig.vsTestVersionDetails.majorVersion + '.' + this.dtaTestConfig.vsTestVersionDetails.minorversion + '.' + this.dtaTestConfig.vsTestVersionDetails.patchNumber;
-        inputDataContract.VsVersionIsTestSettingsPropertiesSupported = this.dtaTestConfig.vsTestVersionDetails.isTestSettingsPropertiesSupported();
-        inputDataContract.MiniMatchTestSourcesFile = this.testSourcesFile;
-        inputDataContract.UseVsTestConsole =  utils.Helper.stringToBool(this.dtaTestConfig.useVsTestConsole);
-        inputDataContract.TestPlatformVersion = this.dtaTestConfig.vsTestVersion;
-        if (utils.Helper.isToolsInstallerFlow(this.dtaTestConfig)) {
-            inputDataContract.COR_PROFILER_PATH_32 = this.dtaTestConfig.toolsInstallerConfig.x86ProfilerProxyDLLLocation;
-            inputDataContract.COR_PROFILER_PATH_64 = this.dtaTestConfig.toolsInstallerConfig.x64ProfilerProxyDLLLocation;
-            inputDataContract.ForcePlatformV2 = true;
-        }
-        // If we are setting the path version is not needed
-        const exelocation = path.dirname(this.dtaTestConfig.vsTestVersionDetails.vstestExeLocation);
-        tl.debug('Adding env var DTA.TestWindow.Path = ' + exelocation);
-        const testWindowRelativeDir = 'CommonExtensions\\Microsoft\\TestWindow';
-        if (exelocation && exelocation.indexOf(testWindowRelativeDir) !== -1) {
-            const ideLocation = exelocation.split(testWindowRelativeDir)[0];
-            tl.debug('Adding env var DTA.VisualStudio.Path = ' + ideLocation);
-            inputDataContract.VisualStudioPath = ideLocation;
-        } else {
-            inputDataContract.VisualStudioPath = exelocation;
-        }
-        inputDataContract.TestWindowPath = exelocation;
-
-        const settingsFile = this.dtaTestConfig.settingsFile;
-        if (utils.Helper.pathExistsAsFile(settingsFile)) {
-            tl.debug('Final runsettings file being used:');
-            utils.Helper.readFileContents(settingsFile, 'utf-8').then(function (settings) {
-                tl.debug('Running VsTest with settings : ');
-                utils.Helper.printMultiLineLog(settings, (logLine) => { console.log('##vso[task.debug]' + logLine); });
-            });
         }
 
         // Pass the acess token as an environment variable for security purposes
-        // let envVars: { [key: string]: string; } = <{ [key: string]: string; }>{};
-        const envVars: { [key: string]: string; } = process.env; // This is a temporary solutio where we are passing parent process env vars, we should get away from this
-        utils.Helper.addToProcessEnvVars(envVars, 'DTA.AccessToken', this.dtaTestConfig.dtaEnvironment.patToken);
+        utils.Helper.addToProcessEnvVars(envVars, 'DTA.AccessToken', this.inputDataContract.AccessToken);
+        this.inputDataContract.AccessToken = null;
```

Comment:
not required?

Reply:
setting it back to null. This is required